classes/Domain/Model/Talk.php

```diff
@@ -25,4 +32,86 @@ public function meta()
     {
         return $this->hasMany(TalkMeta::class, 'talk_id');
     }
+
+    /**
+     * Return a collection of recent talks
+     *
+     * @param $admin_id
+     * @param int $limit
+     *
+     * @return array|Talk[]
+     */
+    public function getRecent($admin_id, $limit = 10)
```

Comment:
For methods like this, use query scopes. Not doing so requires instantiation of an active record when, query scopes are the "factory method" we more or less need. https://laravel.com/docs/5.5/eloquent#local-scopes